### PR TITLE
admin create user password field

### PR DESCRIPTION
**Pull request by arguingcrab**

### CODE REVIEW

**Comment by adriaanwm** [comment]
I believe this breaks the edit-user form because the user form is reused

File: app/views/admin/users.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 from . import bp
 from app import create_logger
-from app.forms.user import UserForm
+from app.forms.user import UserForm, CreateUserForm
```
Comment (adriaanwm) [inline app/views/admin/users.py]: should `UserForm` now be `EditUserForm` to be more explicit?

**Comment by adriaanwm** [comment]
@geekforbrains why did you add "needs review" label here, is that as a reminder to review later, or am I misunderstanding how to use that label.

File: app/views/admin/users.py (hunk 1 above)
Comment (geekforbrains) [reply app/views/admin/users.py]: I don't think we need to rename to `EditUserForm` since create form is the only specific use case.

**Review by geekforbrains — CHANGES_REQUESTED** [review]
Needs rebase/conflicts fixed.

**Review by geekforbrains — APPROVED** [review]

**Pull request merged**